git-secrets-scan-action/README.md

```diff
@@ -0,0 +1,42 @@
+# GitHub Action to `git secrets --scan` in your CI  🔄
+
+This simple action uses the [vanilla AWS CLI](https://docs.aws.amazon.com/cli/index.html) and [git secrets](https://github.com/awslabs/git-secrets) to scan your repository for secrets being commited to public. 
```

Comment:
This GitHub Action doesn't actually use AWS CLI.

Reply:
oh right, missed to remove that